src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/LimitedPartnershipsApiApplication.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.limitedpartnershipsapi;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class LimitedPartnershipsApiApplication {
+    public static void main(String[] args) {
+        SpringApplication.run(LimitedPartnershipsApiApplication.class, args);
+    }
+}
```

Comment:
We shouldn't forget to do this, which is in the equivalent ROE class:

```
    @PostConstruct
    public void init() {
        // This is to prevent times being out of time by an hour during British Summer Time in MongoDB
        // MongoDB stores UTC datetime, and LocalDate doesn't contain timezone
        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
    }
```
    
Suggest adding this now or if not add a task to JIRA so that it's not forgotten.

Reply:
This looks like it relates to mongo but as there is no mongo specific task for the api I can add this tot he next api PR.